tmail-backend/jmap/extensions/src/test/java/com/linagora/tmail/james/jmap/firebase/FirebaseClientTest.java

```diff
@@ -0,0 +1,78 @@
+package com.linagora.tmail.james.jmap.firebase;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+import java.util.Map;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import com.google.firebase.messaging.FirebaseMessagingException;
+import com.linagora.tmail.james.jmap.model.FirebaseToken;
+import com.linagora.tmail.james.jmap.model.MissingOrInvalidFirebaseCredentialException;
+
+@Disabled("manual test suite")
+class FirebaseClientTest {
+    public static final FirebaseToken VALID_TOKEN = new FirebaseToken("change_me");
```

Comment:
```suggestion
    public static final FirebaseToken VALID_TOKEN = new FirebaseToken(Optional.ofNullable(System.getenv("FIREBASE_TOKEN")).orElse("change_me"));
```
easier for developer